src/approaches/online_nsrt_learning_approach.py

```diff
@@ -0,0 +1,143 @@
+"""An NSRT learning approach that collects and learns from data online.
+
+Example command:
+    python src/main.py --approach online_nsrt_learning --seed 0 \
+        --env cover \
+        --max_initial_demos 1 \
+        --num_train_tasks 1000 \
+        --num_test_tasks 10 \
+        --max_num_steps_interaction_request 10 \
```

Comment:
why is this so low?

Reply:
just because each action is a pick or a place in cover